docs/tutorials/FlowTutorial.md

```diff
@@ -0,0 +1,2176 @@
+
+```
+cd src/tap/test/
+openroad -gui
+```
+
+In the `Tcl Commands` section of GUI
+
+```
+source gcd_nangate45.tcl
+```
+
+View the resulting tap cell insertion as follows:
+
+![Tap_Cell_Insertion](./images/tapcell_insertion_view.webp)
+
+### Tie Cells
+
+The tie cell is a standard cell, designed specially to provide the high
+or low signal to the input (gate terminal) of any logic gate.
+Where ever netlist is having any pin connected to 0 logic or 1 logic
+(like .A(1'b0) or .IN(1'b1), a tie cell gets inserted there.
+
+Refer to the following built-in example [here](https://github.com/The-OpenROAD-Project/OpenROAD/blob/master/src/ifp/test/tiecells.tcl)
+to learn about Tie cell insertion.
+
+To check this in OpenROAD tool root directory:
+
+```
+cd src/ifp/test/
+openroad
+source tiecells.tcl
+```
+
+Refer the following verilog code which have tie high/low net.
+[here](https://github.com/The-OpenROAD-Project/OpenROAD/blob/master/src/ifp/test/tiecells.v)
+```
+AND2_X1 u2 (.A1(r1q), .A2(1'b0), .ZN(u2z0));
+AND2_X1 u3 (.A1(u1z), .A2(1'b1), .ZN(u2z1));
+```
+With following `insert_tiecells` command:
+```
+insert_tiecells LOGIC0_X1/Z -prefix "TIE_ZERO_"
+insert_tiecells LOGIC1_X1/Z
+```
+During floorplan stage, those nets converted to tiecells as follows
+based on library(This is Nangate45 specific):
+```
+[INFO IFP-0030] Inserted 1 tiecells using LOGIC0_X1/Z.
+[INFO IFP-0030] Inserted 1 tiecells using LOGIC1_X1/Z.
+```
+
+### Macro or Cell Placement
+
+#### Macro Placement
+
+In this section, you will explore various placement options for macros
+and standard cells and study the impact on area and timing.
+
+Refer to the following built-in example [here](https://github.com/The-OpenROAD-Project/OpenROAD/blob/master/src/gpl/test/macro01.tcl)
+to learn about macro placement.
+
+Placement density impacts how widely standard cells are placed in the
+core area. To view this in OpenROAD GUI run the following command(s) in the terminal in OpenROAD tool root directory:
+
+```
+cd src/gpl/test/
+openroad -gui
+```
+
+In the `Tcl Commands` section of GUI
+
+```
+source macro01.tcl
+```
+
+Read the resulting macro placement with a complete core view:
+
+![Macro Placement full view](./images/macro_place_full_view.png)
+
+Zoomed view of cell placement:
+
+![Macro Placement Zoomed view](./images/macro_place_close_view.png)
+
+Reduce the placement density and observe the impact on placement, by
+running below command in `Tcl Commands` section of GUI:
+
+```
+global_placement -density 0.6
+```
+
+Read the resulting macro placement with a complete core view:
+
+![Placement Density 60](./images/placement_density_06_full.png)
+
+Zoomed view of cell placement:
+
+![Placement density zoomed](./images/placement_density_06_zoomed.png)
+
+##### Macro Placement With Halo Spacing
+
+<!-- Explore macro placement with halo spacing, refer to the example
+[here](../../src/mpl/test/gcd_mem1_02.tcl). -->
```

Comment:
@vvbandeira this script is not there anymore, do I remove this part?

Reply:
@habibayassin 
Remove and update with existing example to match tutorial requirement. Respective images need update.
Also if it's moving permanently to ORFS all the paths should be updated to `../tools/OpenROAD/src/ ` format to access `OpenROAD` examples. Am I right @vvbandeira ?

Reply:
As a first step using the link to the OR submodule is fine, but we should consider further decoupling the tutorial. However, @vijayank88, the script in question was deleted from the OR repo -- let me followup on this.